pkg/arvo/app/chat-hook.hoon

```diff
@@ -327,7 +335,7 @@
       ?+  mark  (on-poke:def mark vase)
           %json         (poke-json:cc !<(json vase))
           %chat-action  (poke-chat-action:cc !<(action:store vase))
-          %noun         (poke-fix-dms:cc %fix-dms)
+          %noun         (poke-noun:cc !<(?(%fix-dm %fix-out-of-sync) vase))
```

Comment:
> `%fix-dms` -> `%fix-dm`

So, the previous fix didn't run in the first place? (^:

Kinda glad tbh, I think this new one fixes most cases and is less invasive.

Reply:
The old one did in fact run, as the arm expected %fix-dms.

Reply:
Yeah, but we were sending a `%fix-dm` instead:

https://github.com/urbit/urbit/blob/5e65abcb7afc3e6c60e67b1f3ea817f606e92a5e/pkg/arvo/app/chat-hook.hoon#L90

Reply:
Yes, but we ignored the contents of the noun.